template/build/webpack.prod.conf.js

```diff
@@ -41,6 +42,22 @@ var webpackConfig = merge(baseWebpackConfig, {
     new ExtractTextPlugin({
       filename: utils.assetsPath('css/[name].[contenthash].css')
     }),
+	new PurifyCSSPlugin({
+        purifyOptions: {
+            whitelist: ['someclass']
+        },
+      // Logs out verbose logs.
+       verbose: true,
+       // An array of file extensions for determining used classes within node_modules
+       moduleExtensions: ['.html', '.vue'],
+       // Give paths to parse for rules. These should be absolute!
+      paths: glob.sync([
+        path.join(__dirname, '../src/components/*.vue'),
```

Comment:
Can make it shorter like this path.join(__dirname, '../**/*.+(html|vue)') Will match subdirectories '/routing/' too, but need match only file those can contain CSS classes.

I added it, tnx for good question.